packages/terra-clinical-result/src/terra-dev-site/doc/ClinicalResult/ClinicalResult.1.doc.mdx

```diff
@@ -0,0 +1,66 @@
+import { Badge } from 'terra-clinical-result/package.json?dev-site-package';
+
+import DefaultResult from '../example/clinical-result/ClinicalResultDefault?dev-site-example';
+import DecoratedResult from '../example/clinical-result/ClinicalResultDecorated?dev-site-example';
+import UnverifiedResult from '../example/clinical-result/ClinicalResultUnverified?dev-site-example';
+import ExtraDisplaysResult from '../example/clinical-result/ClinicalResultExtraDisplays?dev-site-example';
+import TruncatedResult from '../example/clinical-result/ClinicalResultTruncated?dev-site-example';
+import InterpretationResults from '../example/clinical-result/ClinicalResultInterpretation?dev-site-example';
+import SizeChangeResult from '../example/clinical-result/ClinicalResultChangeSize?dev-site-example';
+import KnownNoData from '../example/clinical-result/ClinicalResultKnownNoData?dev-site-example';
+import KnownError from '../example/clinical-result/ClinicalResultKnownError?dev-site-example';
+import AccidentalError from '../example/clinical-result/ClinicalResultAccidentalError?dev-site-example';
+import StatusEnteredInError from '../example/clinical-result/ClinicalResultStatusEnteredInError?dev-site-example';
+
+import ClinicalResultsProps from 'terra-clinical-result/src/ClinicalResult?dev-site-props-table';
+
+<Badge />
+# Terra Clinical Result
+
+The Terra Clinical Result package is a collection of standardized views for presenting clinical results documented to a Patient's Medical Chart,
+such as Vital Signs, Laboratory Results, and Discretely Documented Values. It also includes additional supporting views where clinical results
+are commonly displayed like flowsheets, result detail viewers, etc.
+
+This default Clinical Result component is intended for displaying a single result value.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-result`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
```

Comment:
Is this used?

Reply:
Oops forgot to update.

Reply:
Updated here https://github.com/cerner/terra-clinical/pull/628/commits/913f6ab0620a3d1d8e10b27a48884696f2881ef0